pkg/api/api.go

```diff
@@ -42,6 +43,18 @@ func NewAPI(o Options) (chi.Router, error) {
 		log:     &logger,
 	}
 
+	cors := cors.New(cors.Options{
+		// AllowedOrigins is used instead of AllowOriginFunc as it's faster and we don't need
+		// to allow credentials, so an asterisk works.
+		AllowOriginFunc:  func(r *http.Request, origin string) bool { return true },
+		AllowedMethods:   []string{"GET", "POST", "PUT", "DELETE", "OPTIONS"},
+		AllowedHeaders:   []string{"Accept", "Authorization", "Content-Type"},
+		ExposedHeaders:   []string{"Link"},
+		AllowCredentials: false,
+		MaxAge:           60 * 60, // 1 hour
```

Comment:
Do we need this set to >0? Not sure what we want cached atm

Reply:
It doesn't hurt — if you're sending > 1 event you'll stop doing another OPTIONS request which is good.